job-dsls/jobs/new_downstream_production.groovy

```diff
@@ -34,18 +34,19 @@ def final REPO_CONFIGS = [
         "jbpm"                      : [],
         "kie-jpmml-integration"     : [],
         "droolsjbpm-integration"    : [],
+        "openshift-drools-hacep"    : [],
         //"droolsjbpm-tools"          : [], // no other repo depends on droolsjbpm-tools
-        "kie-uberfire-extensions"   : [],
         "kie-wb-playground"         : [],
-        "kie-wb-common"             : [
-                label                  : "kie-rhel7&&kie-mem24g&&gui-testing"
```

Comment:
are we removing the label for build node ?

Reply:
OK - will re-edit it there - The old FDB (disabled one in /pullrequest folder) has it, the recent one on /upstream-downstream folder hast it not.